Library/Homebrew/version.rb

```diff
@@ -192,11 +192,8 @@ def self.create(val)
   end
 
   def initialize(val)
-    if val.respond_to?(:to_str)
-      @version = val.to_str
-    else
-      raise TypeError, "Version value must be a string; got a #{val.class} (#{val})"
-    end
+    raise TypeError, "Version value must be a string; got a #{val.class} (#{val})" unless val.respond_to?(:to_str)
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.